app/controllers/dashboard/work_versions_controller.rb

```diff
@@ -37,6 +37,9 @@ def show
     def edit
       @work_version = policy_scope(WorkVersion).find(params[:id])
       authorize(@work_version)
+
+      my_creator = Creator.find_or_create_by_user(current_user)
+      @work_version.build_creator_alias(creator: my_creator)
```

Comment:
@awead I thought you'd appreciate the Walt Whitman vibes in this line

Reply:
"Oh captain, my captain!"